ipaserver/install/cainstance.py

```diff
@@ -461,6 +461,8 @@ def configure_instance(self, host_name, dm_password, admin_password,
                 self.step("updating IPA configuration", update_ipa_conf)
                 self.step("enabling CA instance", self.__enable_instance)
                 if not promote:
+                    self.step("exposing CA instance on LDAP",
```

Comment:
I think we need to call this step only when installing a replica (otherwise it will also get executed for first master installation). Not sure which variable to check, though...

Reply:
Ok, @tiran I removed ACK and added WIP.
Let us try to figure out something.

Reply:
There is only one possible path to run into the situation: Install a primary master in DL0 mode. But DL0 installation is deprecated and only used in testing and not supported.

Reply:
Even in DL1 the master install will go through this code path. I just tried to install a master in DL1 and was able to see in the output:
```
  [23/29]: enabling CA instance
  [24/29]: exposing CA instance on LDAP
/usr/lib/python3.6/site-packages/ipaserver/install/cainstance.py:1297: DeprecationWarning: ldap_enable is deprecated, use ldap_configure instead.
  self.ldap_enable('CA', self.fqdn, None, basedn, config)
  [25/29]: migrating certificate profiles to LDAP
```